source/Octopus.Tentacle.Client/TentacleClientOptions.cs

```diff
@@ -6,11 +6,6 @@ namespace Octopus.Tentacle.Client
     public class TentacleClientOptions
     {
         public RpcRetrySettings RpcRetrySettings { get; }
-        
-        /// <summary>
-        /// Disables the use of ScriptServiceV3Alpha, even if it's supported on the Tentacle
-        /// </summary>
-        public bool DisableScriptServiceV3Alpha { get; set; }
```

Comment:
We don't need to disable this as the capabilities service now dictates what services are available.